packages/core/src/libraries/user.ts

```diff
@@ -106,18 +107,45 @@ export const createUserLibrary = (queries: Queries) => {
       { retries, factor: 0 } // No need for exponential backoff
     );
 
+  const updateUserById = async (
```

Comment:
Should put the validPhoneNumber validation logic at the API koaGuard level?

Reply:
I was considering to make this phone number guard at API level. But since the DB insert and update operation could bypass the API-level check since some of them were used in scenarios other than in API handlers.
Make this change on the DB queries/libraries methods IMO should be the safest and the most easy-to-maintain way of doing this.